src/bin/build_octree.rs

```diff
@@ -29,10 +29,24 @@ struct CommandlineArguments {
     /// This decides on the number of bits used to encode each node.
     #[structopt(long = "resolution", default_value = "0.001")]
     resolution: f64,
+    // Flag to skip bbox calculation and use a fixed bbox.
+    // The fixed bbox is extends 6400000 length units in each direction.
```

Comment:
... -> `The fixed bbox extents have 6_400_000 length units in each direction, so they cover the whole earth when using ECEF coordinates.`?